cannot find module npm/bin/npm/npm-cli.js

I cannot get `npm install` to run.

* Windows 10 x64
* Node v5.10.1
* Atom 1.7.3


When I hit <kbd>CTRL</kbd><kbd>ALT</kbd><kbd>B</kbd>, I get this error:

```sh
module.js:341
    throw err;
    ^
 
Error: Cannot find module 'C:\Users\Danny\Dev\handmade-quake-js\node_modules\npm\bin\npm-cli.js'
    at Function.Module._resolveFilename (module.js:339:15)
    at Function.Module._load (module.js:290:25)
    at Function.Module.runMain (module.js:447:10)
    at startup (node.js:146:18)

    at node.js:404:3
```

I made sure the only path I had in `PATH` related to node or npm was my node path.

Any advice on how to get `atom-build-npm-apm` to find my `npm.cmd` file?

--- a/lib/npm-apm.js
+++ b/lib/npm-apm.js
@@ -21,7 +21,6 @@ export function provideBuilder() {
       delete require.cache[realPackage];
       const pkg = require(realPackage);
 
-      const executableExtension = /^win/.test(process.platform) ? '.cmd' : '';
       // https://github.com/mochajs/mocha/issues/1844
       const env = {FORCE_COLOR: '1', MOCHA_COLORS: '1', NPM_CONFIG_COLOR: 'always'};
       const errorMatch = [
@@ -30,7 +29,7 @@ export function provideBuilder() {
       ];
       const config = [ {
         name: pkg.engines && pkg.engines.atom ? 'apm: install' : 'npm: install',
-        exec: (pkg.engines && pkg.engines.atom ? 'apm' : 'npm') + executableExtension,
+        exec: (pkg.engines && pkg.engines.atom ? 'apm' : 'npm'),
         args: [ 'install' ],
         env: env,
         errorMatch: errorMatch,
@@ -41,7 +40,7 @@ export function provideBuilder() {
         if (pkg.scripts.hasOwnProperty(script)) {
           config.push({
             name: 'npm: ' + script,
-            exec: 'npm' + executableExtension,
+            exec: 'npm',
             args: [ 'run', script ],
             env: env,
             errorMatch: errorMatch,

